SDKBuildScripts/Update-PlayFabSdk.ps1

```diff
@@ -0,0 +1,289 @@
+points to a sibling directory to the SDKGeneratory repository called "sdks".
+
+.PARAMETER TargetSource
+Override the target source parameter name passed to the generation script.
+
+.PARAMETER KeepSource
+Indicites whether or not to remove all generated source files from the destination location.
+
+.PARAMETER Beta
+Indicates whether or not to include any APIs tagged with as beta.
+
+.EXAMPLE
+Update-PlayFabSdk -SdkNames CSharpSDK
+
+Run the PlayFab API generation for the CSharp SDK using the API specification documents from
+the PlayFab git repository.
+
+.EXAMPLE
+Update-PlayFabSdk CSharpSDK,JavaScriptSDK -Vertical example
+
+Run the PlayFab API generation for the CSharp and JavaScript SDKs using the API specification
+documents from the example vertical and include all of the beta SDKs.
+
+This is functionally identical to using -ApiSpecPfUrl "https://example.playfabapi.com/apispec"
+
+.EXAMPLE
+Update-PlayFabSdk CSharpSDK,JavaScriptSDK -Vertical example -Cloud sample
+
+Run the PlayFab API generation for the CSharp and JavaScript SDKs using the API specification
+documents from the example vertical and the sample cloud.
+
+This is functionally identical to using -ApiSpecPfUrl "https://example.sample.playfabapi.com/apispec"
+
+.EXAMPLE
+Update-PlayFabSdk UnitySDK -Beta
+
+Run the PlayFab API generation for the CSharp and JavaScript SDKs using the API specification
+documents from the example vertical and include all of the beta SDKs.
+
+#>
+[CmdletBinding(SupportsShouldProcess = $true, DefaultParameterSetName = "ApiSpecGitUrl")]
+param(
+    [Parameter(Position = 0, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory)]
+    [ValidateSet(
+        "ActionScriptSDK",
+        "Cocos2d-xSDK",
+        "CSharpSDK",
+        "JavaSDK",
+        "JavaScriptSDK",
+        "LuaSDK",
+        "NodeSDK",
+        "Objective_C_SDK",
+        "PhpSDK",
+        "PostmanCollection",
+        "PythonSDK",
+        "SdkTestingCloudScript",
+        "UnrealMarketplacePlugin",
+        "UnitySDK",
+        "WindowsSDK",
+        "XPlatCppSDK")]
+    [string[]]$SdkNames,
+    [Parameter(ParameterSetName="ApiSpecPath", ValueFromPipelineByPropertyName = $true)]
+    [AllowEmptyString()]
+    [string]$ApiSpecPath,
+    [Parameter(ParameterSetName="ApiSpecPfUrl", ValueFromPipelineByPropertyName = $true)]
+    [AllowEmptyString()]
+    [string]$ApiSpecPfUrl,
+    [Parameter(ParameterSetName="ApiSpecPfUrlCloudVertical", ValueFromPipelineByPropertyName = $true)]
+    [string]$Vertical = "master",
+    [Parameter(ParameterSetName="ApiSpecPfUrlCloudVertical", ValueFromPipelineByPropertyName = $true)]
+    [string]$Cloud,
+    [Parameter(ParameterSetName="ApiSpecGitUrl", ValueFromPipelineByPropertyName = $true)]
+    [AllowEmptyString()]
+    [string]$ApiSpecGitUrl,
+    [Parameter(ValueFromPipelineByPropertyName = $true)]
+    [string]$OutputPath = "..\..\sdks",
```

Comment:
Traditionally this relative path has been measured from the "Generate.js" file, rather than this script file.
Line 283 makes me I believe that it should be "../sdks" here too.
Please confirm.